pwa/src/templates/pluginsDetailTemplate/PluginsDetailTemplate.module.css

```diff
@@ -153,3 +153,65 @@
 .showmoreVersionsButton {
   text-align: center;
 }
+
+/* readme styling */
+.readme {
+  padding-inline-end: var(--gateway-ui-size-md);
+  padding-block-start: var(--gateway-ui-size-md);
+  padding-block-end: var(--gateway-ui-size-md);
+}
+
+.readme code {
+  background-color: var(--gateway-ui-color-lightgrey);
+  outline: 1px solid var(--gateway-ui-color-grey);
+  padding-inline: var(--gateway-ui-size-2xs);
+  border-radius: var(--gateway-ui-size-2xs);
+}
+.readme pre code {
+  padding-inline: var(--gateway-ui-size-xs);
+  padding-block: var(--gateway-ui-size-xs);
+}
```

Comment:
I think it is better if we change the line height for some code lines.
Right now the code lines are on top of one another

![image](https://user-images.githubusercontent.com/43807324/212902957-6c032bc9-82bc-40b7-9098-7534e7d02d7c.png)
![image](https://user-images.githubusercontent.com/43807324/212902897-958a6773-b480-4ded-9d22-25cff27fe65f.png)